daft/daft/__init__.pyi

```diff
@@ -510,6 +510,7 @@ class S3Config:
         profile_name (str, optional): Name of AWS_PROFILE to load, defaults to None which will then check the Environment Variable `AWS_PROFILE` then fall back to `default`
         multipart_size (int, optional): The size of multipart part (bytes), the size range should be 5MB to 5GB, defaults to 8MB.
         multipart_max_concurrency (int, optional): The max concurrency of upload part per object, defaults to 100.
+        custom_retry_msgs (list[str], optional): Will retry the request if any custom retry message appeared in the error message of response, defaults to None.
```

Comment:
I'm happy with this but have you also considered instead of a `list[str]`, to use a regexp pattern instead?

Reply:
Ok, regex pattern might a good option, but do you mean use a regex pattern list or just a single regex patten? from  user side, regex might a little bit complex than simple keywords, for example, if we want to match both "UnexpectEOF" and "Timeout",  user need to config `(UnexpectEOF|Timeout)` instead of ["UnexpectEOF", "Timeout"], but if the user intention is match `Keyword*`, regex pattern is better, if the user intention is match "UnexpectEOF" and "Timeout" and `Keyword*`, a regex pattern list is better, how do you think?

Reply:
I'm open to either. But I am happy to merge this current implementation in as well if that is sufficient for your use case.

Reply:
thanks @kevinzwang, I think the current implementation is sufficient, thx again